src/mutation/__tests__/RelayMutationQuery-test.js

```diff
@@ -43,6 +43,11 @@ describe('RelayMutationQuery', () => {
       node => !node.getSchemaName || node.getSchemaName() !== 'source';
     return filterRelayQuery(RelayTestUtils.getNode(...args), filterCallback);
   }
+  function getMutationWithoutClientId(mutation) {
```

Comment:
Yes, let's stop adding this field in the plugin rather than filter it in tests. We do not strictly need to fetch the `clientMutationId` field.